src/Microsoft.ML.StandardTrainers/Standard/Online/LinearSvm.cs

```diff
@@ -45,12 +45,12 @@ namespace Microsoft.ML.Trainers
     /// | Required NuGet in addition to Microsoft.ML | None |
     ///
     /// ### Training Algorithm Details
-    /// Linear [SVM](https://en.wikipedia.org/wiki/Support-vector_machine#Linear_SVM) is a trainer that implements
+    /// Linear [SVM](https://en.wikipedia.org/wiki/Support-vector_machine#Linear_SVM) implements
     /// an algorithm that finds a hyperplane in the feature space for binary classification, by solving an [SVM problem](https://en.wikipedia.org/wiki/Support-vector_machine#Computing_the_SVM_classifier).
     /// For instance, with feature values $f_0, f_1,..., f_{D-1}$, the prediction is given by determining what side of the hyperplane the point falls into.
-    /// That is the same as the sign of the feautures' weighted sum, i.e. $\sum_{i = 0}^{D-1} \left(w_i * f_i \right)$, where $w_0, w_1,..., w_{D-1}$ are the weights computed by the algorithm.
+    /// That is the same as the sign of the feautures' weighted sum, i.e. $\sum_{i = 0}^{D-1} \left(w_i * f_i \right) + b$, where $w_0, w_1,..., w_{D-1}$ and $b$ are the weights and bias computed by the algorithm.
     ///
-    /// This algorithm uses the PEGASOS method, which alternates between stochastic gradient descent steps and projection steps,
+    /// This algorithm implemented is the PEGASOS method, which alternates between stochastic gradient descent steps and projection steps,
```

Comment:
>implemented is [](start = 23, length = 14)

implements

Reply:
algorithm cannot implement itself.

---
In reply to: [277888217](https://github.com/dotnet/machinelearning/pull/3551#discussion_r277888217) [](ancestors = 277888217)

Reply:
I'll let @TomFinley comment on this ..

Reply:
I got Tom's support. :)

---
In reply to: [277907721](https://github.com/dotnet/machinelearning/pull/3551#discussion_r277907721) [](ancestors = 277907721)

Reply:
Just to register my own opinion since I was explicitly asked, I agree with @shmoradims that the proposed edit was ungrammatical, and I also agree with what I interpret as @wschin's point that the proposed edit is not only ungrammatical but confuses the issue, and probably should not have happened. So I hope this phrasing remains unchanged, from its original point.